conans/test/functional/toolchains/cmake/test_cmake.py

```diff
@@ -648,7 +648,8 @@ def build(self):
 
         # The create flow must work
         c.run("create . --name=pkg --version=0.1 -pr:b=default -o test*:shared=True")
-        assert "1/1 Test #1: example ..........................   Passed" in c.out
+        assert str(c.out).count("1/1 Test #1: example ..........................   Passed") == 2
+        assert "pkg/0.1: RUN: ctest --build-config Release --parallel"
```

Comment:
Looks like this assertion can never fail, as it should end with `in c.out`.

Thanks for adding this feature though :+1: